src/main/resources/application.yaml

```diff
@@ -78,4 +79,9 @@ identity-account-mapper:
     register-beneficiary: /beneficiary
     add-payment-modality: /paymentModality
     update-payment-modality: /paymentModality
-    account-lookup: /beneficiary
\ No newline at end of file
+    account-lookup: /beneficiary
+mifos-connector:
```

Comment:
these changes should also to be added to the env-template integration test Job, there it should be added with service name instead of ingress URL

Reply:
Yeah once i do the integration as job in template I'll add it